interface/usergroup/facilities_add.php

```diff
@@ -16,6 +16,7 @@
 <link rel="stylesheet" type="text/css" href="<?php echo $GLOBALS['webroot'] ?>/library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" />
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
 <script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-min-1-9-1/index.js"></script>
+<script src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery-migrate-1.2.1.min.js"></script> 
```

Comment:
hi @visolve-openemr ,
Why is this jquery-migrate-1.2.1.min.js script needed here?